integration/spark/src/main/java/org/apache/carbondata/spark/load/CarbonLoaderUtil.java

```diff
@@ -215,6 +227,105 @@ public static void executeGraph(CarbonLoadModel loadModel, String storeLocation,
             info, loadModel.getPartitionId(), loadModel.getCarbonDataLoadSchema());
   }
 
+  public static void executeNewDataLoad(CarbonLoadModel loadModel, String storeLocation,
```

Comment:
Can we move this class or this function out of carbon-spark package? It is a utility function class for loading, should not depend on spark, right?